Applications/NodeConf-TLP.md

```diff
@@ -0,0 +1,74 @@
+## Introduction & History
+
+NodeConf was started in 2011 by Mikeal Rogers (@mikeal) and Chris Williams (@voodootikigod).
+
+Today there are several NodeConf's all run by differnet organizers who loosely support each other in the `nodeconf` GitHub org.
+
+## Scope
+
+All current and future NodeConf's:
```

Comment:
I want to leave the door open for people bringing conferences in that aren't called NodeConf. If we end up building a support system for these events I'd like it to be as widely accessible as possible.